public/javascripts/index.js

```diff
@@ -41,17 +28,39 @@ const Letter = React.createClass({
 	    </div>
```

Comment:
I have a question about this return. Why is there just a div closing tag? I suppose you will fill in information for this letter?